rllib/evaluation/postprocessing.py

```diff
@@ -39,10 +39,7 @@ def compute_advantages(rollout: SampleBatch,
             processed rewards.
     """
 
-    traj = {}
-    trajsize = len(rollout[SampleBatch.ACTIONS])
-    for key in rollout:
-        traj[key] = np.stack(rollout[key])
+    rollout_size = len(rollout[SampleBatch.ACTIONS])
```

Comment:
We shouldn't copy the entire batch here. Significantly speeds up postprocessing.